js/bitget.js

```diff
@@ -2797,9 +2797,10 @@ module.exports = class bitget extends Exchange {
         return response;
     }
 
-    async withdraw (code, amount, address, tag = undefined, params = {}, chain) {
+    async withdraw (code, amount, address, tag = undefined, params = {}) {
         await this.loadMarkets ();
-        this.checkAddress (address);
+        this.checkAddress(address);
+        const chain = params.chain
```

Comment:
```suggestion
        const { chain, ...params } = params
```
だか
```
delete params.chain //書き方あってるかわからない（オブジェクトのプロパティ消すやつ
```
だかした方がいいかもです
paramsに余計なのが入ったまま使われてしまうので